pythran/pythonic/include/types/numpy_expr.hpp

```diff
@@ -428,6 +428,9 @@ namespace pythonic
 
       numpy_expr(Args const &... args);
 
+      numpy_expr& operator=(numpy_expr const&) = default;
+      numpy_expr& operator=(numpy_expr&&) = default;
```

Comment:
I'm dubious about these ones. numpy_expr sometimes have a reference as member variable, which make the move operator a non sense.
The situation you're meeting, with a forward declaration of the type, is rather tricky. the only pythranic way to do this would be to force the type to be assignable, so that you get an ndarray to hold the result of the expression.